nav2_rviz_plugins/src/navigation_dialog.cpp

```diff
@@ -117,8 +117,8 @@ NavigationDialog::startNavigation(double x, double y, double theta, std::string
   auto is_action_server_ready = action_client_->wait_for_action_server(std::chrono::seconds(5));
   if (!is_action_server_ready) {
     RCLCPP_ERROR(client_node_->get_logger(), "NavigateToPose action server is not available."
-      " Is the initial pose set?");
-    return;
+      " Check if Gazebo simulator is running and the initial pose is set");
```

Comment:
This isn't for just gazebo, please remove the gazebo language.

Reply:
I agree it can be any simulator or any real robot. I will change it to "Check if the simulator or the robot is running. Make sure the initial pose is set." instead.

Reply:
That works, I think the "Is XYZ running?" is sufficient knowing that in the log statement will list the process in it but is probably good to add more language for new users